examples/with-tailwindcss/styles/index.css

```diff
@@ -1,23 +1,18 @@
-@import './button.css';
-
 @tailwind base;
 @tailwind components;
-@tailwind utilities;
+
+.btn-blue {
+  @apply bg-blue-500 text-white font-bold py-2 px-4 rounded;
+}
 
 .hero {
-  width: 100%;
-  color: #333;
+  @apply py-20;
 }
 
 .title {
-  margin: 0;
-  width: 100%;
```

Comment:
`@tailwind base` resets the margin on heading elements, so `margin: 0` isn't necessary. Removing `width: 100%` had no effect, so I kept it out.